src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -95,14 +97,15 @@ private void submitInner(FlightContext flightContext)
         "INSERT INTO "
             + FLIGHT_TABLE
             + " (flightId, submit_time, class_name, status, stairway_id, debug_info)"
-            + "VALUES (:flightId, CURRENT_TIMESTAMP, :className, :status, :stairwayId, :debugInfo)";
```

Comment:
One interesting learning here: if we're going to generate the timestamp on the host anywhere (which we do elsewhere so that we can keep it in sync between `flightlog|flight` and `flightworking`), we should do it everywhere.

I discovered this because it's apparently possible to have clock skew between the Linux VM that Docker runs containers on in MacOS and the MacOS system clock, which lead to unit test scenarios where `submit_time` got ahead of `completed_time` (which obviously broke test invariants).

Reply:
I'd prefer to use `CURRENT_TIMESTAMP` and generate it in the database.  I did it like this as a first pass to avoid plumbing [this overload](https://docs.oracle.com/javase/7/docs/api/java/sql/Connection.html#prepareStatement(java.lang.String,%20java.lang.String[]))  through `NamedParameterPreparedStatement` without some discussion.  But it sounds like if we go with a unique ID key in `flightlog` this might be moot...

Reply:
Restored original behavior.